cmd/delete/account/account.go

```diff
@@ -31,7 +31,7 @@ func NewaccountCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "account",
 		Short: "delete account",
```

Comment:
**Short** should be `Deletes an Account`, please check and update this as well in all the commands wherever required